modules/standard/Subprocess.chpl

```diff
@@ -1043,17 +1067,19 @@ module Subprocess {
     the child process. See :proc:`subprocess.send_signal`.
    */
   proc subprocess.abort() throws {
+    use Sys only SIGABRT;
```

Comment:
I'd say `import Sys` instead as slightly "cleaner", here an in other similar cases.
However `use only` works fine, too.